run.sh

```diff
@@ -55,6 +56,9 @@ ref_gff_path="$ref_gff"
 if is_url "$ref_fasta"; then
     download_and_decompress "$ref_fasta" "./$target_dir"
     ref_fasta_path="./$target_dir/$(basename "$ref_fasta")"
+    if [[ ${ref_fasta_path: -3} == ".gz" ]]; then
```

Comment:
What do we gain by removing the ".gz" at the end?

Reply:
After execute the download_and_decompress "$ref_fasta" "./$target_dir", we have Mus_musculus.GRCm38.dna.toplevel.fa.gz and Mus_musculus.GRCm38.dna.toplevel.fa . Since $ref_fasta_path is the filename passed in app.py, it needs to be modified to remove the .gz. Then OS will use Mus_musculus.GRCm38.dna.toplevel.fa instead of Mus_musculus.GRCm38.dna.toplevel.fa.gz to pass it into reform.py.

Reply:
but how has it been working all this time?

Reply:
reform.py doesn't take in compressed files. Maybe we can update the reform.py to do so at https://github.com/gencorefacility/reform

Reply:
We have in fact already extracted the compressed files, but the filenames have not been changed yet, so this line works on changing the filenames.